common.gradle

```diff
@@ -0,0 +1,107 @@
+apply plugin: 'application'
+apply plugin: 'java-library'
+
+sourceCompatibility = 8
+targetCompatibility = 8
+
+def rapidwrightDir = System.getenv("RAPIDWRIGHT_PATH")
```

Comment:
```suggestion
def rapidwrightDir = System.getenv('RAPIDWRIGHT_PATH') == null ? buildscript.sourceFile.getParent() : System.getenv("RAPIDWRIGHT_PATH")
```
Perhaps something like this might work?

Reply:
Good idea. I've now removed the requirement for `RAPIDWRIGHT_PATH` here, but it is required for other parent gradles that source `common.gradle`.